src/v/redpanda/admin/debug.cc

```diff
@@ -810,6 +817,79 @@ admin_server::get_partition_state_handler(
     co_return ss::json::json_return_type(std::move(response));
 }
 
+ss::future<ss::json::json_return_type>
+admin_server::get_producers_state_handler(
+  std::unique_ptr<ss::http::request> req) {
+    const model::ntp ntp = parse_ntp_from_request(req->param);
+    auto timeout = std::chrono::duration_cast<model::timeout_clock::duration>(
+      10s);
+    auto result = co_await _tx_gateway_frontend.local().get_producers(
```

Comment:
should we check here if the frontend is initialized (as is done for the other endpoint)?

Reply:
I don't think we need to because it is started before creating server, I added just in case.